code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm

```diff
@@ -139,10 +142,10 @@
 	return ..()
 
 /mob/living/simple_animal/hostile/abnormality/hatred_queen/AttackingTarget()
-	return OpenFire()
+	return OpenFire(target)
```

Comment:
This is not needed. Target is a simple mob variable, you don't actually have to put it in any proc most of the time.